packages/thumbprint-react/components/Alert/test.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { BannerAlert, InPageAlert } from './index';
+
+describe('BannerAlert', () => {
+    test('renders messageType good', () => {
+        const wrapper = mount(
+            <BannerAlert messageType="good">
+                <p>Alert content</p>
+            </BannerAlert>,
+        );
+        expect(
+            wrapper
+                .find('div')
+                .at(0)
+                .hasClass('goodNews'),
+        ).toBe(true);
```

Comment:
Personally I think the snapshot tests are good enough for this kind of test. @lavelle, curious to know what you think.

Reply:
Agree that any tests like this that are simply asserting the static state of the DOM after rendering are better covered by snapshots. Reserve your custom unit tests for testing dynamic, interactive behaviour, or the logic of different functions and conditionals.